src/components/ui/Accordion/stories/Accordion.stories.tsx

```
@@ -103,22 +103,33 @@ export const OpenMultiple: Story = {
             <>
                 <Button onClick={() => setMultiple(!multiple)}>{`Toggle Open Multiple (${multiple ? 'on' : 'off'})`}</Button>
-                <Button onClick={() => setValue([])}>Close All</Button>
-                <Button onClick={() => setValue([1])}>Open 2</Button>
-                <Button onClick={() => setValue([0])}>Open 0</Button>
-                <Button onClick={() => setValue([0, 1])}>Open 0, 1</Button>
-                <AccordionExample value={value} onValueChange={setValue} openMultiple={multiple} />
+                <Button onClick={() => setValue(multiple ? [] : '')}>Close All</Button>
+                <Button onClick={() => setValue(multiple ? [1] : 1)}>Open 2</Button>
+                <Button onClick={() => setValue(multiple ? [0] : 0)}>Open 0</Button>
+                <Button onClick={() => setValue(multiple ? [0, 1] : 0)}>Open 0, 1</Button>
+                {multiple ? (
```

ESLint

[error] Expected newline between test and consequent of ternary expression.

(multiline-ternary)